CHANGELOG.md

```diff
@@ -1,28 +1,136 @@
+## [15.9.0](https://github.com/commercetools/merchant-center-application-kit/compare/v15.9.0...v15.8.0) (2020-01-23)
+
+#### 🚀 Type: New Feature
+
+- `application-shell`
+  - [#1249](https://github.com/commercetools/merchant-center-application-kit/pull/1249) feat: send performance timing values to GTM ([@emmenko](https://github.com/emmenko))
+
+#### 🐛 Type: Bug
+
+- `permissions`
+
+  - [#1259](https://github.com/commercetools/merchant-center-application-kit/pull/1259) fix(permissions): data fences to be overwritten by manage resource permission ([@tdeekens](https://github.com/tdeekens))
+
+Note that in this release we have added log statements whenever we encounter an what we consider an not recommended use of our permission components. The permission components include `RestrictedByPermissions`, `injectAuthorized` and `useIsAuthorized`.
```

Comment:
```suggestion
Note that in this release, whenever permissions are evaluated, we might log some warnings in case the demanded permissions do not follow our recommended approach (read below). The components affected by this check include `RestrictedByPermissions`, `injectAuthorized` and `useIsAuthorized`.
```